unit-files/ping-pong-sk@.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description="Sidekick for Ping-Pong"
+BindsTo=ping-pong@%i.service
+After=ping-pong@%i.service
+
+[Service]
+EnvironmentFile=/etc/environment
+RemainAfterExit=yes
+
+ExecStart=/bin/sh -c \
+  "sleep 1; \
+  port=$(/usr/bin/docker inspect ping-pong-%i | jq -r '.[].NetworkSettings.Ports.\"8880/tcp\"[].\"HostPort\"'); \
+  echo $port; \
```

Comment:
You should write clear log messages, not just print data. otherwise you might get confused at some point when under pressure what this value is you see in the logs.